040-SDK/010-TypeScript/020-configuration.mdx

```diff
@@ -38,6 +38,24 @@ const xata = new XataClient({ apiKey: XATA_API_KEY });
 
 In case your runtime does not provide a built-in Fetch API (such as with versions prior to Node 18), you will need to pass a fetch implementation to the `fetch ` constructor parameter of the XataClient in the `xata.ts` / `xata.js `file in your project.
 
+## Passing options to fetch
+
+Some frameworks, such as Next.js, support additional fetch options. These settings allow you to specifically control how caching and revalidation are handled for each fetch operation on the server.
+
+The Xata SDK supports the `fetchOptions` parameter for passing options to fetch.
+
+This parameter can be used to configure cache revalidation at a timed interval:
```

Comment:
I'd make this more clear it's only for Next.js

Reply:
Will add clarifications once we have findings from https://github.com/xataio/client-ts/issues/1311 as it doesn't seem to actually work as documented, maybe there's more steps needed in Next.js.